kodexa/platform/client.py

```diff
@@ -475,9 +475,10 @@ def taxonomies(self) -> ProjectTaxonomiesEndpoint:
 
     @property
     def assistants(self) -> ProjectAssistantsEndpoint:
-        url = f"/api/projects/{self.id}/assistants"
-        response = self.client.get(url)
-        return [AssistantEndpoint.parse_obj(assistant).set_client(self.client) for assistant in response.json()]
+        return ProjectAssistantsEndpoint().set_client(self.client).set_project(self)
+        #url = f"/api/projects/{self.id}/assistants"
```

Comment:
Its best not to leave code commented out - since it can be confusing if you are searching or quickly reviewing code.  We have Git so It's just as easy to delete it and use VCS as needed.